specification/iotoperations/IoTOperations.Management/main.tsp

```diff
@@ -119,3 +128,64 @@ interface DataflowEndpoint {
   delete is ArmResourceDeleteWithoutOkAsync<DataflowEndpointResource>;
   listByResourceGroup is ArmResourceListByParent<DataflowEndpointResource>;
 }
+
+// /instance/dataflowProfiles/dataflowGraphs resource Operations
+
+@armResourceOperations
+@added(Versions.`2025-07-01-preview`)
+interface DataflowGraph {
+  get is ArmResourceRead<DataflowGraphResource>;
+  createOrUpdate is ArmResourceCreateOrUpdateAsync<DataflowGraphResource>;
+  delete is ArmResourceDeleteWithoutOkAsync<DataflowGraphResource>;
+  listByDataflowProfile is ArmResourceListByParent<DataflowGraphResource>;
+}
+
+// /instance/registryEndpoints resource Operations
+@armResourceOperations
+@added(Versions.`2025-07-01-preview`)
+interface RegistryEndpoint {
+  get is ArmResourceRead<RegistryEndpointResource>;
+  createOrUpdate is ArmResourceCreateOrUpdateAsync<RegistryEndpointResource>;
+  delete is ArmResourceDeleteWithoutOkAsync<RegistryEndpointResource>;
+  listByInstanceResource is ArmResourceListByParent<RegistryEndpointResource>;
+}
+
+// /instance/diagnostics resource Operations
+@armResourceOperations
+@added(Versions.`2025-07-01-preview`)
+interface Diagnostic {
```

Comment:
Vasco has told me they don't expect for this to be ready for 2507. They'd like for it to be in an internal private preview state by then. We can sync to discuss what that means but for now let's assume we won't be adding this resource type to the 2025-07-01-preview version